.github/workflows/build-themes.yml

```
@@ -0,0 +1,38 @@
+name: Build themes
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-themes:
+    name: Build Storefront Themes
+    runs-on: ubuntu-latest
+    if: |
+      github.event.head_commit.message != 'Initial commit' &&
+      !contains(github.event.head_commit.message, '[skip ci]')
```

Não tem motivo pra executar o workflow em todo commit, só a cada release novo do Storefront, certo?

```suggestion
on: registry_package

jobs:
  build-themes:
    name: Build Storefront themes
    runs-on: ubuntu-latest
```